commisery/cli/__init__.py

```diff
@@ -175,10 +175,12 @@ def _read_commit_message():
 
         answers = inquirer.prompt(questions)
 
-        answers["scope"] = answers.get("scope") if answers.get("scope") else None
+        answers["scope"] = answers.get("scope") if answers.get("scope", None) else None
```

Comment:
Inquirer returns an empty string (`""`) in case the entry is skipped, which will result in a check failure (scope is not allowed to be empty (`""`))